actions/databases.go

```diff
@@ -0,0 +1,290 @@
+package actions
+
+import (
+	"errors"
+	"log"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gobuffalo/buffalo"
+)
+
+// DatabasesList gets a list of databases for a given account
+//   If the `all=true` parameter is passed it will return a list of clusters in addition to instances.
+func DatabasesList(c buffalo.Context) error {
+	// if all param is given, we'll return information about both instances and clusters
+	// otherwise, only database instances will be returned
+	all := false
+	if b, err := strconv.ParseBool(c.Param("all")); err == nil {
+		all = b
+	}
+
+	rdsClient := RDS[c.Param("account")]
```

Comment:
this still bugs me 😆 👯‍♂️

Reply:
> i don't think we want that check in each handler
i do actually 🤣

Reply:
is it for readability? the middleware will make sure we don't accidentally forget to check that.

Reply:
yeah i shouldn't have to jump around between files, etc to understand that the code won't panic if that account doesn't exist.  it's a nit pick 💯 but my eyes go right to it 😆 .

Reply:
ok, i can see that. i'll get rid of that middleware

Reply:
🙏